src/slider.jsx

```diff
@@ -307,6 +307,18 @@ const Slider = React.createClass({
   },
 
 
+  // Needed to prevent text selection when dragging the slider handler.
+  // In the future, we should consider use <input type="range"> to avoid
+  // similar issues.
+  _toggleSelection(value) {
+    let body = document.getElementsByTagName('body')[0];
+    body.style['user-select'] = value;
```

Comment:
I think that we should have beend using     AutoPrefix.set(style, 'transform', 'scale(' + this.props.minScale + ')');